www/json/connectionConfig.json

```diff
@@ -1,5 +1,5 @@
 {
-    "connectUrl": "http://52.91.65.149:8080",
+    "connectUrl": "http://52.91.65.149:80",
```

Comment:
nit" technically, you don't need to specify the port, because the default port for http is 80